benchmark_test.go

```diff
@@ -63,7 +65,7 @@ func BenchmarkContainerStart(b *testing.B) {
 		b.Error(err)
 		return
 	}
-	spec, err := GenerateSpec(ctx, client, nil, WithImageConfig(image), withTrue())
+	spec, err := GenerateSpec(ctx, client, &containers.Container{ID: b.Name()}, WithImageConfig(image), withTrue())
```

Comment:
what's the significance of adding an "empty" container with the name filled in for ID? I only ask because our examples still show `nil` here and other than being passed to the options helpers, I don't see what `GenerateSpec` does with it.

Reply:
it panics

Reply:
Caught my eyes too, I think it was only because it's part of `SpecOpt` signature. Maybe we should replace it with ID directly there.

Reply:
other spec opts use this for more info, like rootfs and snapshotter

Reply:
of course it panics.. my eyes were obviously not following the code path :) Anyway..I'm fine generating a separate PR to update the getting started and README.md examples that are passing `nil` if that's what we want to do.

Reply:
Updated the getting started doc